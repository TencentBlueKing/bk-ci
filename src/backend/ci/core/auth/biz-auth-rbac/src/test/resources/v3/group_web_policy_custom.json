{
  "type": "group_web_policy",
  "project_id": "bkdevops",
  "subject": {
    "type": "group",
    "id": "14608",
    "name": "v3-104-web迁移"
  },
  "permissions": [
    {
      "actions": [
        {
          "id": "project_view"
        }
      ],
      "resources": [
        {
          "type": "project",
          "paths": [
            [
              {
                "id": "bkdevops",
                "name": "bkdevops",
                "system_id": "bk_ci",
                "type": "project"
              }
            ]
          ]
        }
      ]
    },
    {
      "actions": [
        {
          "id": "rule_enable"
        }
      ],
      "resources": [
        {
          "type": "rule",
          "paths": [
            [
              {
                "id": "bkdevops",
                "name": "bkdevops",
                "system_id": "bk_ci",
                "type": "project"
              }
            ]
          ]
        }
      ]
    },
    {
      "actions": [
        {
          "id": "project_views_manager"
        }
      ],
      "resources": [
        {
          "type": "project",
          "paths": [
            [
              {
                "id": "bkdevops",
                "name": "bkdevops",
                "system_id": "bk_ci",
                "type": "project"
              }
            ]
          ]
        }
      ]
    },
    {
      "actions": [
        {
          "id": "pipeline_view"
        }
      ],
      "resources": [
        {
          "type": "pipeline",
          "paths": [
            [
              {
                "id": "bkdevops",
                "name": "bkdevops",
                "system_id": "bk_ci",
                "type": "project"
              }
            ]
          ]
        }
      ]
    },
    {
      "actions": [
        {
          "id": "repertory_view"
        }
      ],
      "resources": [
        {
          "type": "repertory",
          "paths": [
            [
              {
                "id": "bkdevops",
                "name": "bkdevops",
                "system_id": "bk_ci",
                "type": "project"
              }
            ]
          ]
        }
      ]
    }
  ],
  "members": [
    {
      "pk": 10468,
      "type": "user",
      "id": "567",
      "expired_at": 1698808387,
      "created_at": "2023-05-05T03:13:09Z"
    }
  ]
}
