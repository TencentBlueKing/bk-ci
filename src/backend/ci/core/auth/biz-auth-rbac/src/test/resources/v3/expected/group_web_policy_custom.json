[ {
  "system" : "bk_ci_rbac",
  "actions" : [ {
    "id" : "project_view"
  } ],
  "resources" : [ {
    "system" : "bk_ci_rbac",
    "type" : "project",
    "paths" : [ [ {
      "system" : "bk_ci_rbac",
      "type" : "project",
      "id" : "bkdevops",
      "name" : "bkdevops"
    } ] ]
  } ]
}, {
  "system" : "bk_ci_rbac",
  "actions" : [ {
    "id" : "rule_enable"
  } ],
  "resources" : [ {
    "system" : "bk_ci_rbac",
    "type" : "rule",
    "paths" : [ [ {
      "system" : "bk_ci_rbac",
      "type" : "project",
      "id" : "bkdevops",
      "name" : "bkdevops"
    } ] ]
  } ]
}, {
  "system" : "bk_ci_rbac",
  "actions" : [ {
    "id" : "pipeline_view"
  } ],
  "resources" : [ {
    "system" : "bk_ci_rbac",
    "type" : "pipeline",
    "paths" : [ [ {
      "system" : "bk_ci_rbac",
      "type" : "project",
      "id" : "bkdevops",
      "name" : "bkdevops"
    } ] ]
  } ]
}, {
  "system" : "bk_ci_rbac",
  "actions" : [ {
    "id" : "repertory_view"
  } ],
  "resources" : [ {
    "system" : "bk_ci_rbac",
    "type" : "repertory",
    "paths" : [ [ {
      "system" : "bk_ci_rbac",
      "type" : "project",
      "id" : "bkdevops",
      "name" : "bkdevops"
    } ] ]
  } ]
} ]
